packages/foreman/foreman/foreman.spec

```diff
@@ -569,6 +570,7 @@ Requires: %{?scl_prefix_ror}rubygem(execjs) < 3.0
 Requires: %{?scl_prefix_ror}rubygem(uglifier) >= 1.0.3
 Requires: %{?scl_prefix_ror}rubygem(sass-rails) >= 5.0
 Requires: %{?scl_prefix_ror}rubygem(sass-rails) < 6.0
+Requires: %{?scl_prefix_ror}rubygem(sass-rails) < 6
```

Comment:
that half-duplicates the line before?

Reply:
Yes, this is a side effect of the pinning (https://github.com/theforeman/foreman/blob/7c26a522e240798e01ec8f9cec3a73b7cc683018/bundler.d/assets.rb#L8):
```ruby
gem 'sass-rails', '~> 5.0', ((RUBY_VERSION < '2.4') ? '< 5.0.8' : '< 6')
```

Would have been more logical if it was:
```ruby
gem 'sass-rails', '>= 5.0', ((RUBY_VERSION < '2.4') ? '< 5.0.8' : '< 6.0')
```

Reply:
https://github.com/theforeman/foreman/pull/6996